source/default-conf.py

```diff
@@ -65,7 +65,7 @@
     'podman-docs'     : ('https://docs.podman.io/en/latest/%s', None),
     'podman-git'      : ('https://github.com/containers/podman/%s', None),
     'docker-docs'     : ('https://docs.docker.com/%s', None),
-    'openshift-docs'  : ('https://docs.openshift.com/container-platform/4.11/%s', None),
+    'openshift-docs'  : ('https://docs.openshift.com/container-platform/4.13/%s', None),
```

Comment:
Might as well? Too bad Red Hat docs don't have an equivalent of `latest`.

Reply:
No harm in bumping!